qa/L0_logging/test.sh

```diff
@@ -588,12 +588,24 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+set +e
+
+FORMAT_TEST_LOG="./log_format_test.log"
+
+python3 -m pytest --junitxml=log_format_test.xml log_format_test.py > $FORMAT_TEST_LOG 2>&1
+
+if [ $? -ne 0 ]; then
+    cat $FORMAT_TEST_LOG
+    echo -e "\n***\n*** Test Failed\n***"
```

Comment:
Can be follow-up but should specify which subtest failed for easier debugging and automation of parsing log errors, ex:
```suggestion
    echo -e "\n***\n*** Log Format Test Failed\n***"
```

Reply:
less of an issue here as they are also picked up in CI as individual xunit style tests